sys/androidmedia/gstamcvideodechybris.c

```diff
@@ -1817,6 +1817,8 @@ static GstFlowReturn
 gst_amc_video_dec_handle_frame (GstVideoDecoder * decoder,
     GstVideoCodecFrame * frame)
 {
+  /* 1 sec is enough for the first hybris input buffer to appear */
```

Comment:
Change this comment to read: "1 sec is enough for the first video input buffer from the hybris layer to appear"
This will add greater clarity to it for someone not familiar with the codebase.